src/node_javascript.cc

```diff
@@ -23,7 +25,24 @@ using v8::String;
 NODE_NATIVES_MAP(V)
 #undef V
 
+static MaybeLocal<String> MaybeLoadSourceFromDisk(Environment* env,
+                                                  const uint8_t* name) {
+  if (!internal_modules_source_dir) {
```

Comment:
I have a mild preference for writing this out as `if (internal_modules_source_dir == nullptr) {` to make it clear that it's a pointer and not a boolean.  Not a deal breaker, though.